packages/genesys-spark-components/src/genesys-spark-utils/get-closest-element.ts

```diff
@@ -0,0 +1,24 @@
+//https://inindca.atlassian.net/browse/COMUI-2673
```

Comment:
Might be worth adding a bit of context to this. Also we should consider marking these type of comments as TODO: so we can use the highlighting feature in VSCode. https://marketplace.visualstudio.com/items?itemName=wayou.vscode-todo-highlight

Reply:
Ah ok, there's already a couple in v3 and a whole bunch in v4 formatted this way. Should we advise against that then?